package.json

```
@@ -18,6 +18,7 @@
     "build": "pnpm --filter node-playground build",
     "lint": "biome check .",
     "lint:fix": "biome check --write .",
+    "ci:lint": "biome ci --formatter-enabled=true --organize-imports-enabled=true --linter-enabled=true --reporter=github",
```

```suggestion
    "ci:lint": "biome ci --formatter-enabled=true --organize-imports-enabled=true  --reporter=github",
```

The option is superfluous, linting is already enabled in the configuration file